code/FulpstationCode/fulp_mobs/apostle.dm

```diff
@@ -0,0 +1,274 @@
+/mob/living/simple_animal/hostile/megafauna/apostle
+	name = "apostle"
+	desc = "The heavens' wrath. You might've fucked up real bad to summon one."
+	health = 1500
+	maxHealth = 1500
+	attack_verb_continuous = "purges"
+	attack_verb_simple = "purge"
+	attack_sound = 'sound/magic/mm_hit.ogg'
+	icon = 'icons/Fulpicons/64x64.dmi'
+	icon_state = "angel"
+	icon_living = "angel"
+	health_doll_icon = "angel"
+	faction = list("apostle")
+	friendly_verb_continuous = "stares down"
+	friendly_verb_simple = "stare down"
+	speak_emote = list("proclaims")
+	armour_penetration = 0
+	melee_damage_lower = 20
+	melee_damage_upper = 20
+	speed = 3
+	move_to_delay = 8
+	ranged = TRUE
+	pixel_x = -16
+	base_pixel_x = -16
+	pixel_y = -16
+	base_pixel_y = -16
+	del_on_death = TRUE
+	//crusher_loot = list()
+	loot = list(/obj/item/gun/magic/staff/chaos) // To be replaced later.
+	gps_name = "Pure Signal"
+	deathmessage = "evaporates in a moment, leaving heavenly light and feathers behind."
+	deathsound = 'sound/tegu_sounds/mob/apostle_death.ogg'
+	attack_action_types = list(/datum/action/innate/megafauna_attack/holy_revival,
+							   /datum/action/innate/megafauna_attack/fire_field)
+	small_sprite_type = /datum/action/small_sprite/megafauna/tegu/angel
+	var/holy_revival_cooldown = 20 SECONDS
+	var/holy_revival_cooldown_base = 20 SECONDS
+	var/holy_revival_damage = 10 // Amount of damage OR heal, depending on target.
```

Comment:
might want to increase this or make them attack faster

Reply:
It's being increased with each conversion, but I suppose it can be a bit faster.

Reply:
First kill with no buddies is probably the hardest

Reply:
Changed in update-v3